All-in-One-fallbacks-Nginx/server.json

```diff
@@ -4,26 +4,53 @@
   },
   "inbounds": [
     {
+      "tag": "Vless-TCP-XTLS",
       "port": 443, // This is TLS entrypoint. This entrypoint does the SSL Termination then routes the request based on the Path or ALPN type.
-      "protocol": "trojan",
+      "protocol": "vless",
       "settings": {
         "clients": [
           {
-            "password": "desdemona99",
+            "id": "90e4903e-66a4-45f7-abda-fd5d5ed7f797",
             "flow": "xtls-rprx-direct" //To enable XTLS Direct, this configuration must be added; otherwise, delete it. V2Ray has deleted the XTLS application since version v4.33.0. If you want to use this application, it is recommended to choose Xray.
```

Comment:
we would recommend new configs to use vision instead of direct for the flow

Reply:
Good to know. Switched to Vision Flow.